OutOfSchool/OutOfSchool.DataAccess/Models/Configurations/QuartzJobConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace OutOfSchool.Services.Models.Configurations;
+
+internal class QuartzJobConfiguration : IEntityTypeConfiguration<QuartzJob>
+{
+    public void Configure(EntityTypeBuilder<QuartzJob> builder)
+    {
+        builder.HasKey(x => x.Id);
+
+        builder.HasIndex(x => x.IsDeleted);
```

Comment:
do we ready need to track deleted quartz jobs? They they are either useful and exist in db or not usefull and we can safely delete it completely.

Reply:
Ok. I'll fix it in the next PR.